frontend/lib/src/models/spending_breakdown.dart

```diff
@@ -0,0 +1,49 @@
+// To parse this JSON data, do
+//
+//     final spendingBreakdown = userFromJson(jsonString);
+
+import 'dart:convert';
+
+import 'package:frontend/src/enums/transaction_enum.dart';
+
+SpendingBreakdown userFromJson(String str) =>
+    SpendingBreakdown.fromJson(json.decode(str));
+
+String userToJson(SpendingBreakdown data) => json.encode(data.toJson());
+
+class SpendingBreakdown {
+  String accountId;
+  TransactionCategories category;
+  double totalStartingBalance;
+  double totalTransactionAmount;
+  DateTime latestTransactionDate;
+  String diffForHumans;
```

Comment:
pls use the website and not copy paste from models, there are user references here